lib/aur-sync

```diff
@@ -205,9 +206,10 @@ while true; do
     shift
 done
 
-tmp=$(mktemp -d --tmpdir "aurutils-$argv0.XXXXXXXX") || exit
-tmp_view=$(mktemp -d --tmpdir "aurutils-$argv0-view.XXXXXXXX") || exit
+mkdir -p "$XDG_RUNTIME_DIR"
 
+tmp=$(mktemp -d --tmpdir "aurutils-$argv0.XXXXXXXX") || exit
+tmp_view=$(mktemp -d "$XDG_RUNTIME_DIR/aurutils/view.XXXXXXXX") || exit
```

Comment:
I don't think this will work, you need to `mkdir -p $XDG_RUNTIME_DIR/aurutils` above or maybe just use `aurutils-view.XXXXXXXX` instead of subfolders `aurutils/view.XXXXXXXX`, depending if you still want to "hide" the folders from a higher level:

```
$ echo $XDG_RUNTIME_DIR
/run/user/1000

$ mktemp -d "$XDG_RUNTIME_DIR/aurutils/view.XXXXXXXX"
mktemp: failed to create directory via template ‘/run/user/1000/aurutils/view.XXXXXXXX’: No such file or directory
```

Reply:
Yeah I did `mkdir -p $XDG_RUNTIME_DIR/aurutils` in the end 🙂